### DISCUSSION TITLE
Can I get a file only with OID as SHA256 signature?

### DISCUSSION THREAD

**Discussion by Dhyey15** [git-lfs/git-lfs#4700] [Q&A]
Hi, I am working on a package in which the .nc (netCDF) files I need seem to be stored on the Git LFS server. Opening the file as .txt I am only able to get its OID as SHA256 signature. I don't have knowledge about its repository. Is there a way I can access the actual file with just knowing OID? 

Thanks,
Dhyey.

**Comment by Cactusinhand** [comment]
Yes, you can use `git cat-file` command to see the file content by providing its OID, like: `git cat-file -p OID`

**Reply by bk2204** [reply]
That's the case for Git OIDs, but not for Git LFS.  For accessing Git LFS objects, you need both the OID and the size.

You can speak [the API protocol](https://github.com/git-lfs/git-lfs/tree/main/docs/api) to the remote, but if you don't know the remote URL, then you won't be able to get the data, since it probably isn't stored on your system.
